internal/config/config.go

```diff
@@ -173,6 +181,15 @@ func bindEnvVars(v *viper.Viper, prefix string, field reflect.StructField) {
 	v.MustBindEnv(key)
 }
 
+func (c *Config) validate() (err error) {
+	if c.Version != "" {
+		if strings.TrimSpace(c.Version) != "1.0" {
```

Comment:
id like to make this smarter in the future, potentially using some form of semver parsing (to two places `x.x`), wyt?

Reply:
Yeah, that could be good. Like a way to ratchet up minor version changes in the config.